### ISSUE TITLE
Enhance interface to display jump fuel

### ISSUE THREAD

**Issue by IntegratedQuantum** [finite-galaxy/finite-galaxy#27]
It would be useful to have the information how much fuel is needed per jump with the best installed drive somewhere in the shipyard and outfitter.

**Comment by finite-galaxy** [comment]
Unfortunately it's a bit complicated; e.g. “best drive” depends on the specific route taken.
To give players an approximate idea, the ship info panel does display the ship's mass; if you're using a hyperdrive or scram drive, the fuel per jump is equal to this; if you're using a jump drive, it's mass +1000. You have to add up the values for fighters, though.

**Comment by finite-galaxy** [comment]
That means one line for each drive; currently there are only three types, but it's not inconceivable more will be added later (or in plug-ins); hard-coding specific outfits is not necessarily a good idea.

**Comment by IntegratedQuantum** [comment]
Isn't there a way to see which outfits are drives and find out their specific fuel values?

**Comment by finite-galaxy** [comment]
Yes, there is the `"jump fuel"` attribute. However, `attributeValues.push_back(Format::Round(fullMass + attributes.Get("jump fuel")));` won't be always correct, because the ship info display simply adds up all values for the ship and all its outfits, regardless whether they're used or not; so if you install two jump drives, it adds +2000, even if you have an ordinary hyperdrive too.
It's possible I'm overlooking something, of course. If you can think of a clever solution, feel free to make a pull request.
